app/models/sync_from_disk.rb

```diff
@@ -44,9 +44,25 @@ def import_and_sync!(deleted: [], changed: [])
   def import!
     raise "Path bad" unless File.exist?(notebook_path)
 
+    # if we have a notebook.yaml file, we're probably dealing with a normal
```

Comment:
I like this, thanks!
I look forward to deleting my (boilerplate) notebook.yaml.